unit/solvers/bdd/miniBDD/miniBDD.cpp

```diff
@@ -188,9 +188,10 @@ SCENARIO("miniBDD", "[core][solver][miniBDD]")
 {
   GIVEN("A bdd for x&!x")
   {
+    null_message_handlert mh;
```

Comment:
You could as well use the common `null_message_handler` available in unit tests.